### PR TITLE
OPath fix

### PR DESCRIPTION
**Pull request by dariomws306**
https://github.com/MicrosoftDocs/office-docs-powershell/issues/4206

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws306 Thank you for the initiative. Much appreciated.

File: docset/windows/addsadministration/Get-ADReplicationSite.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ This command gets all Active Directory Replication sites.
 
 ### Example 2: Get replication sites by flag
 ```
-PS C:\> Get-ADReplicationSite -Properties * -Filter {WindowsServer2003KCCSiteLinkBridgingEnabled -eq $TRUE}
+PS C:\> Get-ADReplicationSite -Properties * -Filter "WindowsServer2003KCCSiteLinkBridgingEnabled -eq `$TRUE"
```
Comment (garycentric) [inline docset/windows/addsadministration/Get-ADReplicationSite.md]: This line seems to be missing a backtick.

File: docset/windows/addsadministration/Move-ADDirectoryServer.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ This command moves the domain controller USER01-DC2 to the site Branch-Office-Si
 
 ### Example 2: Move read-only domain controllers to an existing site
 ```
-PS C:\> Get-ADDomainController -Filter {IsReadOnly -eq $True} | Move-ADDirectoryServer -Site "RODC-Site-Name"
+PS C:\> Get-ADDomainController -Filter "IsReadOnly -eq `$True" | Move-ADDirectoryServer -Site "RODC-Site-Name"
```
Comment (garycentric) [inline docset/windows/addsadministration/Move-ADDirectoryServer.md]: This line seems to be missing a backtick.

File: docset/windows/addsadministration/Remove-ADClaimType.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ This command removes the claim type with the name Title.
 
 ### Example 2: Get all disabled claim types and remove them
 ```
-PS C:\> Get-ADClaimType -Filter { Enabled -eq $FALSE } | Remove-ADClaimType
+PS C:\> Get-ADClaimType -Filter "Enabled -eq `$False" | Remove-ADClaimType
```
Comment (garycentric) [inline docset/windows/addsadministration/Remove-ADClaimType.md]: This line seems to be missing a backtick.

File: docset/winserver2012-ps/activedirectory/Get-ADClaimTransformPolicy.md [hunk 4]
```diff
@@ -193,37 +193,37 @@ Get-ADComputer -Filter *
...
 
 To get all groups that have a group category of Security and a group scope of Global, use one of the following commands:
 
-Get-ADGroup  -filter {GroupCategory  -eq "Security"  -and GroupScope -eq "Global"}
+Get-ADGroup -Filter "GroupCategory -eq 'Security' -and GroupScope -eq 'Global'"
 
 -or-
 
-Get-ADGroup -filter {GroupType -band 0x80000000}
+Get-ADGroup -Filter 'GroupType -band 0x80000000"
```
Comment (garycentric) [inline docset/winserver2012-ps/activedirectory/Get-ADClaimTransformPolicy.md]: This line has an opening backtick but closes with a quotation mark.

File: docset/winserver2012-ps/activedirectory/Get-ADDomainController.md [hunk 5]
```diff
@@ -189,7 +189,7 @@ Get all global catalogs in a given site.
 
 ### -------------------------- EXAMPLE 13 --------------------------
 ```
-C:\PS>Get-ADDomainController -Server "research.fabrikam.com" -Filter { isGlobalCatalog -eq $true -and isReadOnly -eq $true }
+C:\PS>Get-ADDomainController -Server "research.fabrikam.com" -Filter "isGlobalCatalog -eq `$true -and isReadOnly -eq `$true"
```
Comment (garycentric) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainController.md]: This line seems to be missing backticks.

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 6]
```diff
@@ -53,7 +53,7 @@ This command gets from an RODC domain controller password replication policy the
 
 ### Example 2: Get the password replication policy allowed list from all RODCs in the domain
 ```
-C:\PS>Get-ADDomainController -Filter {IsReadOnly -eq $true} | Get-ADDomainControllerPasswordReplicationPolicy -Allowed
+C:\PS>Get-ADDomainController -Filter "IsReadOnly -eq `$true" | Get-ADDomainControllerPasswordReplicationPolicy -Allowed
```
Comment (garycentric) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicy.md]: This line seems to be missing a backtick.

File: docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicyUsage.md [hunk 7]
```diff
@@ -62,7 +62,7 @@ The command displays the name and object class of each account returned.
 
 ### Example 3: Get a list of accounts cached across all RODCs
 ```
-PS C:\> Get-ADDomainController -Filter {IsReadOnly -eq $true} | Get-ADDomainControllerPasswordReplicationPolicyUsage
+PS C:\> Get-ADDomainController -Filter "IsReadOnly -eq `$true" | Get-ADDomainControllerPasswordReplicationPolicyUsage
```
Comment (garycentric) [inline docset/winserver2012-ps/activedirectory/Get-ADDomainControllerPasswordReplicationPolicyUsage.md]: This line seems to be missing a backtick.

File: docset/winserver2012-ps/activedirectory/Get-ADReplicationSite.md [hunk 8]
```diff
@@ -48,7 +48,7 @@ Get all the sites.
 
 ### -------------------------- EXAMPLE 2 --------------------------
 ```
-C:\PS>Get-ADReplicationSite -Properties * -Filter {WindowsServer2003KCCSiteLinkBridgingEnabled -eq $TRUE}
+C:\PS>Get-ADReplicationSite -Properties * -Filter "WindowsServer2003KCCSiteLinkBridgingEnabled -eq `$True"
```
Comment (garycentric) [inline docset/winserver2012-ps/activedirectory/Get-ADReplicationSite.md]: This line seems to be missing a backtick.

File: docset/winserver2012-ps/activedirectory/Move-ADDirectoryServer.md [hunk 9]
```diff
@@ -58,7 +58,7 @@ Move the domain controller "FABRIKAM-DC2" to the site "Branch-Office-Site".
 
 ### -------------------------- EXAMPLE 2 --------------------------
 ```
-C:\PS>Get-ADDomainController -Filter {IsReadOnly -eq $true} | Move-ADDirectoryServer -site "RODC-Site-Name"
+C:\PS>Get-ADDomainController -Filter "IsReadOnly -eq `$true" | Move-ADDirectoryServer -site "RODC-Site-Name"
```
Comment (garycentric) [inline docset/winserver2012-ps/activedirectory/Move-ADDirectoryServer.md]: This line seems to be missing a backtick.

File: docset/winserver2012-ps/activedirectory/Remove-ADClaimType.md [hunk 10]
```diff
@@ -39,7 +39,7 @@ Remove the claim type with the name 'Title'.
 
 ### -------------------------- EXAMPLE 2 --------------------------
 ```
-C:\PS>Get-ADClaimType -Filter { Enabled -eq $FALSE } | Remove-ADClaimType
+C:\PS>Get-ADClaimType -Filter "Enabled -eq `$False" | Remove-ADClaimType
```
Comment (garycentric) [inline docset/winserver2012-ps/activedirectory/Remove-ADClaimType.md]: This line seems to be missing a backtick.

File: docset/winserver2012r2-ps/addsadministration/Get-ADDomainControllerPasswordReplicationPolicy.md [hunk 11]
```diff
@@ -61,7 +61,7 @@ This command gets from an RODC domain controller password replication policy the
 
 ### Example 2: Get the password replication policy allowed list from all RODCs in the domain
 ```
-C:\PS>Get-ADDomainController -Filter {IsReadOnly -eq $true} | Get-ADDomainControllerPasswordReplicationPolicy -Allowed
+C:\PS>Get-ADDomainController -Filter "IsReadOnly -eq `$true" | Get-ADDomainControllerPasswordReplicationPolicy -Allowed
```
Comment (garycentric) [inline docset/winserver2012r2-ps/addsadministration/Get-ADDomainControllerPasswordReplicationPolicy.md]: This line seems to be missing a backtick.

File: docset/winserver2012r2-ps/addsadministration/Get-ADDomainControllerPasswordReplicationPolicyUsage.md [hunk 12]
```diff
@@ -70,7 +70,7 @@ The command displays the name and object class of each account returned.
 
 ### Example 3: Get a list of accounts cached across all RODCs
 ```
-PS C:\> Get-ADDomainController -Filter {IsReadOnly -eq $true} | Get-ADDomainControllerPasswordReplicationPolicyUsage
+PS C:\> Get-ADDomainController -Filter "IsReadOnly -eq `$true" | Get-ADDomainControllerPasswordReplicationPolicyUsage
```
Comment (garycentric) [inline docset/winserver2012r2-ps/addsadministration/Get-ADDomainControllerPasswordReplicationPolicyUsage.md]: This line seems to be missing a backtick.

File: docset/winserver2012r2-ps/addsadministration/Get-ADReplicationSite.md [hunk 13]
```diff
@@ -52,7 +52,7 @@ This command gets all Active Directory Replication sites.
 
 ### Example 2: Get replication sites by flag
 ```
-PS C:\>Get-ADReplicationSite -Properties * -Filter {WindowsServer2003KCCSiteLinkBridgingEnabled -eq $TRUE}
+PS C:\>Get-ADReplicationSite -Properties * -Filter "WindowsServer2003KCCSiteLinkBridgingEnabled -eq `$True"
```
Comment (garycentric) [inline docset/winserver2012r2-ps/addsadministration/Get-ADReplicationSite.md]: This line seems to be missing a backtick.

File: docset/winserver2012r2-ps/addsadministration/Move-ADDirectoryServer.md [hunk 14]
```diff
@@ -58,7 +58,7 @@ This command moves the domain controller USER01-DC2 to the site Branch-Office-Si
 
 ### Example 2: Move read-only domain controllers to an existing site
 ```
-PS C:\>Get-ADDomainController -Filter {IsReadOnly -eq $true} | Move-ADDirectoryServer -site "RODC-Site-Name"
+PS C:\>Get-ADDomainController -Filter "IsReadOnly -eq `$true" | Move-ADDirectoryServer -site "RODC-Site-Name"
```
Comment (garycentric) [inline docset/winserver2012r2-ps/addsadministration/Move-ADDirectoryServer.md]: This line seems to be missing a backtick.

File: docset/winserver2012r2-ps/addsadministration/Remove-ADClaimType.md [hunk 15]
```diff
@@ -43,7 +43,7 @@ This command removes the claim type with the name Title.
 
 ### Example 2: Get all disabled claim types and remove them
 ```
-PS C:\>Get-ADClaimType -Filter { Enabled -eq $FALSE } | Remove-ADClaimType
+PS C:\>Get-ADClaimType -Filter "Enabled -eq $False" | Remove-ADClaimType
```
Comment (garycentric) [inline docset/winserver2012r2-ps/addsadministration/Remove-ADClaimType.md]: Based on other command lines in this PR, I wonder if **$False** should be enclosed in backticks here.

**Comment by garycentric** [comment]
@dariomws306 

In reviewing the diffs for this PR, it seems that several of the command lines are missing backticks and/or quotation marks. I've added comments to the lines that looked to me like they needed review. 

I'm returning this PR to you for review. Please describe the resolution, if any is necessary, for the next PR reviewer.

Thanks,

Gary

**Comment by RAJU2529** [comment]
@garycentric. any I get the docs Microsoft content , so that I can follow-up if I able to

File: docset/windows/addsadministration/Get-ADReplicationSite.md (hunk 1 above)
Comment (dariomws306) [reply docset/windows/addsadministration/Get-ADReplicationSite.md]: Thank you!

**Comment by dariomws306** [comment]
> @dariomws306
> 
> In reviewing the diffs for this PR, it seems that several of the command lines are missing backticks and/or quotation marks. I've added comments to the lines that looked to me like they needed review.
> 
> I'm returning this PR to you for review. Please describe the resolution, if any is necessary, for the next PR reviewer.
> 
> Thanks,
> 
> Gary

Thank you @garycentric. I have reviewed all of them and only 2 were really wrong: single quotation mark instead double and forgot to add a backtick before a $false system variable.

When you enclose OPath filters between quotation marks, you should add a backtick before system variables, not enclosed them.
More information:
https://docs.microsoft.com/powershell/exchange/exchange-server/recipient-filters/recipient-filters#additional-opath-syntax-information

**Pull request merged**